code/modules/mob/living/carbon/xenomorph/castes/behemoth/abilities_behemoth.dm

```diff
@@ -117,19 +110,31 @@
 	span_xenonotice(BEHEMOTH_ROCK_EATING_MESSAGES), null, 5)
 	return TRUE
 
+/// Enables or disables connections used to check for climbing and other stuff.
+/obj/structure/xeno/earth_pillar/proc/handle_connections(toggle)
```

Comment:
Its not a big deal but 'toggle' as an arg is not really clear. What you're actually doing is turning it ON or OFF. Toggle implies TRUE means its swapping state regardless of what it currently is, which its not doing.